app/views/notifications/index.html.erb

```diff
@@ -19,25 +19,42 @@
 
       <% elsif @notifications.present? && current_user.user_type == "tasker" %>
         <div class="notification-feed col s8">
+
           <% @notifications.each do |object| %>
             <div class="viewed feed">
               <img src="<%= object.sender.image_url %>" alt="" />
               <p class="title"> <%= object.message %> </p>
-              <a href="#modal1" onclick="modal_object(<%= object.sender.to_json %>, <%= object.id %>)" class="btn info_btn modal-trigger"> view information</a>
+              <%if object.notifiable_type == 'Quote'%>
+              <a href="#quote_modal" onclick="quote_object(<%= object.sender.to_json %>,<%=object.notifiable.to_json%>, <%= object.id %>)" class="btn info_btn modal-trigger"> view quote</a>
+
+              <%else%>
+
+              <%#= render partial: "show_information" %>
+              <%end%>
 
             </div>
-          <% end %>
-          <%#= render partial: "show_information" %>
-          <div id="modal1" class="modal info_modal">
-              <div class="modal-content">
-                <h4>Tasker Information</h4>
-                <p class="artisan_name">Name: <span id="name"></span></p>
-                <p class="artisan_phone">Phone: <span id="phone"></span></p>
-                <p class="artisan_email">Email: <span id="email"></span></p>
+
+            <div id="quote_modal" class="modal info_modal">
+                <div class="modal-content">
+                  <h4>Quote Information</h4>
+                  <p class="artisan_name">Name: <span id="name"></span></p>
+                  <h3 class="">Quote: <span id="quote"></span></h3>
+
+                  <div id= "information">
+                    <h5>Details</h5>
+                    <p class="artisan_phone">Phone: <span id="phone"></span></p>
+                    <p class="artisan_email">Email: <span id="email"></span></p>
+                    <a href="#!" class=" modal-action modal-close waves-effect waves-green btn-flat">Close</a>
+                  </div>
+
+                  <button id='quote_accept' class="btn teal modal-action" onclick="quotation('accepted',<%=object.notifiable.id%>)">Accept</button>
```

Comment:
There should be appropriate spacing, like so
`onclick="quotation('accepted', <%= object.notifiable.id %>)"`
here and other places.